lib/quant/indicators/indicator.rb

```diff
@@ -193,67 +202,14 @@ def input
         t0.send(source)
       end
 
-      # def warmed_up?
-      #   true
-      # end
-
-      # attr_reader :dc_period
-
-      # def points_for(series:)
-      #   @points_for_cache[series] ||= self.class.new(series:, settings:, cloning: true).tap do |indicator|
-      #     series.ticks.each { |tick| indicator.points.push(tick.indicators[self]) }
-      #   end
-      # end
-
-      # # Ticks belong to the first series they're associated with always
-      # # NOTE: No provisions for series merging their ticks to one series!
-      # def parent_series
-      #   series.ticks.empty? ? series : series.ticks.first.series
-      # end
-
-      # # Returns the last point of the current indicator rather than the entire series
-      # # This is used for indicators that depend on dominant cycle or other indicators
-      # # to compute their data points.
-      # def current_point
-      #   points.size - 1
-      # end
-
-      # def dominant_cycles
-      #   parent_series.indicators.dominant_cycles
-      # end
-
-      # # Override this method to change source of dominant cycle computation for an indicator
-      # def dominant_cycle_indicator
-      #   @dominant_cycle_indicator ||= dominant_cycles.band_pass
-      # end
-
-      # def ensure_not_dominant_cycler_indicator
-      #   return unless is_a? Quant::Indicators::DominantCycles::DominantCycle
-
-      #   raise 'Dominant Cycle Indicators cannot use the thing they compute!'
-      # end
-
-      # # Returns the dominant cycle point for the current indicator's point
-      # def current_dominant_cycle
-      #   dominant_cycle_indicator[current_point]
-      # end
```

Comment:
**comment**:  All of this has been incorporated/implemented and can be dropped now (this was notes from original implementation).